Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -698,16 +698,20 @@ def get_file_data(self, file_hash):
         cmd_url = f'/files/{file_hash}'
         return self.ms_client.http_request(method='GET', url_suffix=cmd_url)
 
-    def list_indicators(self, indicator_id: Optional[str] = None) -> List:
+    def list_indicators(self, indicator_id: Optional[str] = None, page_size: str = '50') -> List:
         """Lists indicators. if indicator_id supplied, will get only that indicator.
 
         Args:
             indicator_id: if provided, will get only this specific id.
+            page_size: specify the page size of the result set.
 
         Returns:
             List of responses.
         """
+        results = {}
+        odata = f'$top={page_size}'
```

Comment:
please use the `params` attribute of `_http_request()` instead of building the url query param string